packages/lib-webpack/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@openshift/dynamic-plugin-sdk-webpack",
+  "version": "1.0.0-alpha1",
+  "description": "Allows building plugins with webpack",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openshift/dynamic-plugin-sdk.git",
+    "directory": "packages/lib-webpack"
+  },
+  "files": [
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "prepack": "yarn build",
+    "prepublish": "yarn test",
+    "build": "rm -rf dist && rollup -c",
+    "lint": "echo 'Run the linter!'",
+    "test": "echo 'Write some tests!'"
+  },
+  "peerDependencies": {
+    "@openshift/dynamic-plugin-sdk": "^1.0.0",
+    "webpack": "^5.69.1"
+  },
+  "dependencies": {
+    "glob": "^7.2.0",
+    "lodash-es": "^4.17.21",
+    "yup": "^0.32.11"
+  },
+  "devDependencies": {
+    "@monorepo/common": "0.0.0-fixed",
+    "@openshift/dynamic-plugin-sdk": "link:../lib-core",
```

Comment:
Once we publish the core SDK package to npmjs, we switch to a semver pattern here.

In general, each `lib-*` package is meant to be versioned and published separately.

Reply:
Is this something we can automate or in some way not miss (a checklist) when publishing? Because I'm pretty sure the npm publish won't resolve this for us, right?

Reply:
@andrewballantyne `devDependencies` part of a package manifest is irrelevant when publishing that package to the npm registry. Even if your app or plugin's dependency has `devDependencies` in its package manifest, it's not taken into account.

Check my other comment https://github.com/openshift/dynamic-plugin-sdk/pull/28#discussion_r816931623

Reply:
This is basically us saying "when building `lib-webpack` we want to compile using the local `lib-core` code present in the monorepo".

Reply:
Doh, yes of course. Sorry about that. I didn't see devDependencies lol.